relay/src/cli.rs

```diff
@@ -119,6 +120,7 @@ pub fn extract_config_env_vars() -> OverridableConfig {
         outcome_source: None, //already extracted in params
         shutdown_timeout: env::var("SHUTDOWN_TIMEOUT").ok(),
         instance: env::var("RELAY_INSTANCE").ok(),
+        server_name: env::var("RELAY_SERVER_NAME").ok(),
```

Comment:
We could default it to `HOSTNAME`.

Something like this (untested):
```rs
env::var("RELAY_SERVER_NAME").or_else(|| env::var("RELAY_SERVER_NAME")).ok()
```

Reply:
I think you made a typo there

Reply:
Updated, copy paste, but get the idea, check `RELAY_SERVER_NAME` then fallback to `HOSTNAME`

Reply:
Does the `HOSTNAME` variable have any special meaning/are we already using it somewhere?

Reply:
A lot of (all?) kubernetes distributions set the `HOSTNAME` variable by default. Also e.g. [bash sets it](https://www.gnu.org/software/bash/manual/bash.html#Bash-Variables).